l1-contracts/src/core/ProofCommitmentEscrow.sol

```diff
@@ -103,9 +97,8 @@ contract ProofCommitmentEscrow is IProofCommitmentEscrow {
    *          The prover must have sufficient balance
    *          The prover's balance will be reduced by the bond amount
    */
-  function stakeBond(uint256 _amount, address _prover) external override onlyRollup {
+  function stakeBond(address _prover, uint256 _amount) external override onlyRollup {
```

Comment:
Natspec should have the params etc. But think we are generally missing some of that. So seems like something that could be acceptable for now, and then lets create a big ass "update natspec" issue.

Created #8912